components/Payment/interfaces/fee_calculator_interfaces.rst

```diff
@@ -0,0 +1,26 @@
+Fee Calculator Interfaces
+=========================
+
+Fee Calculator Interface
+------------------------
+
+This interface will ask you to implement the following methods:
+
++-------------------------------------------------------------------+--------------------------------------------------------+---------------+
+| Method                                                            | Description                                            | Returned type |
++===================================================================+========================================================+===============+
+| calculate(PaymentSubjectInterface $payment, array $configuration) | Calculate the subject's fee according to configuration | int           |
++-------------------------------------------------------------------+--------------------------------------------------------+---------------+
+| getType()                                                         | Get calculator's type                                  | string        |
++-------------------------------------------------------------------+--------------------------------------------------------+---------------+
+
+Delegating Fee Calculator Interface
+-----------------------------------
+
+This interface will ask you to implement the following methods:
+
++---------------------------------------------+--------------------------------------------------------+---------------+
+| Method                                      | Description                                            | Returned type |
++=============================================+========================================================+===============+
+| calculate(PaymentSubjectInterface $payment) | Calculate the subject's fee according to configuration | int           |
```

Comment:
Again, ``Calculate`` => ``Calculates``, be aware of this grammar detail, that is obviously not... obvious for no-native ;)